Source/ablastr/coarsen/average.cpp

```diff
@@ -105,7 +105,13 @@ namespace ablastr::coarsen::average
                                            "source MultiFab and destination MultiFab have different IndexType");
 
         // Number of guard cells to fill on coarse patch and number of components
-        const amrex::IntVect ngrow = (mf_src.nGrowVect() + 1) / crse_ratio;
+    amrex::IntVect ngrow;
+
+        for ( int l = 0; l < AMREX_SPACEDIM; ++l ) {
+            if ( crse_ratio[l] == 1 ) ngrow[l] = mf_src.nGrowVect()[l]; // no coarsening
+            else              ngrow[l] = (mf_src.nGrowVect()[l] + 1) / crse_ratio[l];
+        }
+
```

Comment:
Thank you, @prkkumar! Would this work as well?
```suggestion
        amrex::IntVect ngrow = mf_src.nGrowVect();
        for (int l=0; l < AMREX_SPACEDIM; ++l) {
             if (crse_ratio[l] > 1)  ngrow[l] = (ngrow[l] + 1) / crse_ratio[l];
        }

```

Reply:
Yes, thank you!